applications/asset_tracker_v2/src/modules/Kconfig.util_module

```diff
@@ -10,13 +10,14 @@ menuconfig UTIL_MODULE
 
 if UTIL_MODULE
 
-config REBOOT_TIMEOUT
```

Comment:
Is a symbol rename really needed ?
And if it is, could the old name be deprecated first to give users time to adjust.

Reply:
No, it's not strictly needed. I removed the name change for now.